docs/source/markdown/podman-remote.1.md

```diff
@@ -17,7 +17,9 @@ Podman uses Buildah(1) internally to create container images. Both tools share i
 (not container) storage, hence each can use or manipulate images (but not containers)
 created by the other.
 
-Podman-remote provides a local client interacting with a Podman backend node through a RESTful API tunneled through a ssh connection. In this context, a Podman node is a Linux system with Podman installed on it and the API service activated. Credentials for this session can be passed in using flags, environment variables, or in `podman-remote.conf`
+Podman-remote provides a local client interacting with a Podman backend node through a RESTful API tunneled through a ssh connection. In this context, a Podman node is a Linux system with Podman installed on it and the API service activated. Credentials for this session can be passed in using flags, environment variables, or in `containers.conf`.
+
+The `containers.conf` file has to be placed under `$HOME/.config/containers/containers.conf` on Linux and Mac and `%APPDATA%\containers\containers.conf` on Windows.
```

Comment:
The below is the placement for rootless, not rootful no?
```suggestion
The `containers.conf` file should be placed under `$HOME/.config/containers/containers.conf` on Linux and Mac and `%APPDATA%\containers\containers.conf` on Windows.
```

Reply:
For podman-remote, shouldn't it always be HOME? Who would possibly want to run podman-remote as root?

Reply:
I think its same for rootless and rootful on mac/windows since its just the client

Reply:
There is no special reason why I did this. Should I include the root paths?

Reply:
>Should I include the root paths?

Oh please no. It's meaningless for windows/mac, and for anyone running podman-remote as root on linux, they just deserve what they get.

Reply:
You don't need the root paths, this is correct

Reply:
@edsantiago that's why I was suggesting the change to should, I wasn't advocating adding the root location too.  I'm OK with it as is.